core/server/common/src/main/java/alluxio/master/journal/raft/RaftJournalSystem.java

```diff
@@ -535,16 +537,11 @@ public synchronized void losePrimacy() {
       return;
     }
     mTransferLeaderAllowed.set(false);
-    try {
       // Close async writer first to flush pending entries.
-      mAsyncJournalWriter.get().close();
-      mRaftJournalWriter.close();
-    } catch (IOException e) {
-      LOG.warn("Error closing journal writer: {}", e.toString());
-    } finally {
-      mAsyncJournalWriter.set(null);
-      mRaftJournalWriter = null;
-    }
+    mAsyncJournalWriter.get().close();
```

Comment:
does these close method throw any runtime exception? Since the original behavior is we still want to set null even after we have exception.

Reply:
+1. To stay on the safe side i recommend we don't catch any exception but still use `finally`:
```
try {
  mAsyncJournalWriter.get().close();
  mRaftJournalWriter.close();
} finally {
  mAsyncJournalWriter.set(null);
  mRaftJournalWriter = null;
}
```